server/configmodule.py

```diff
@@ -14,7 +14,8 @@ class Config:
     VERSION = '{}-{}'.format(os.environ.get('WEBSITE_HASH'),
                              os.environ.get('MIXER_HASH'))
     API_ROOT = 'http://127.0.0.1:8081'  # Port for Kubernetes ESP.
-    GCS_BUCKET = os.environ.get('GCS_BUCKET') or ''
+    # TODO: Use prod recon api in prod.
+    RECON_API_URL = 'https://autopush.recon.datacommons.org/id/resolve'
```

Comment:
@Spaceenter can we use prod recon service for prod website now?

Reply:
+1 to using prod here.

The recon service also uses the BT cache, and stores a version.

Suppose we make a DCID map change, what's the rollout process?  For example, during go/dcrelease, should we do something like this:

1. Send PR to update the cache version in recon service
2. Push to staging recon service
3. Validate staging website (which uses staging.recon)
4. Push to prod recon

Reply:
Updated mixer to serve recon APIs here as well. So there is no special handling for the recon APIs.